lib/browser/api/in-app-purchase.js

```diff
@@ -0,0 +1,46 @@
+'use strict'
+
+const binding = process.atomBinding('in_app_purchase')
+const v8Util = process.atomBinding('v8_util')
+
+module.exports = {
+
+  canMakePayments: function() {
+    return binding.canMakePayments();
+  },
+
+  getReceiptURL: function() {
+    return binding.getReceiptURL();
+  },
+
+  purchaseProduct: function(productID, quantity, callback) {
+
+    if (typeof productID !== 'string') {
+      throw new TypeError('productID must be a string')
+    }
+
+    if (typeof quantity !== 'number') {
+      quantity = 1
+    }
+
+    if (typeof callback !== 'function') {
+      callback = function() {};
```

Comment:
Please handle absence of a callback on the native side.
It doesn't make much sense to create and call an empty function.